### PR TITLE
fixed bad spacing format

### PR DESCRIPTION
**Pull request by jamesmandrews**
Noticed that 2 `end` statements were in alignment, so I found the corresponding if statement and fixed the alignment.

### CODE REVIEW

**Comment by TayMcKenzieNZ** [comment]
Thanks 🙏

File: client/Expressions.lua [hunk 1]
```diff
@@ -39,34 +39,34 @@ if Config.ExpressionsEnabled then
 
 
     -- Load the expression once the player has spawned. Standalone, QBCore and ESX --
```
Comment (iSentrie) [inline client/Expressions.lua:41]: why left this?

Comment (TayMcKenzieNZ) [reply client/Expressions.lua:41]: Huh?

Comment (iSentrie) [reply client/Expressions.lua:41]: nvm, i just woke up

Comment (TayMcKenzieNZ) [reply client/Expressions.lua:41]: Ok...

**Pull request merged**